media/test_flows/flow_quick_replies.json

```diff
@@ -0,0 +1,105 @@
+{
+  "base_language": "eng",
+  "action_sets": [
+    {
+      "y": 2,
+      "x": 100,
+      "destination": "55a4abc9-1556-4bb5-8005-a23532108f55",
+      "uuid": "b9494431-b00c-4be3-9413-387ec69a8eb4",
+      "actions": [
+        {
+          "quick_replies": {
+            "por": [
+              {
+                "title": "Op\u00e7\u00e3o 1"
+              },
+              {
+                "payload": "2"
+              }
+            ],
+            "eng": [
+              {
+                "payload": "1",
+                "title": "Option 1"
```

Comment:
Currently, we prepared the code to receive payload as response back to RapidPro. So why wouldn't be it a good practice to do that? We think that has utility when, for example, you want to add emojis to display to the user and to send to RapidPro something different.